test/dx_mgn_pool.js

```diff
@@ -400,17 +460,39 @@ contract("DxMgnPool", (accounts) => {
 
       await truffleAssert.reverts(instance.triggerMGNunlockAndClaimTokens(), "Pooling period is not yet over.")
     })
+    it("can deal with the case, totalDeposit == 0 (e.g. all funds are lost in dutchX)", async () => {
```

Comment:
"can deal with the case" is not a really descriptive behaviour. Can you clarify what the expected behaviour is? Also "funds are lost" sounds a bit like we are hacked. Maybe "balance of deposits is 0"?